src/components/Categories.jsx

```diff
@@ -1,8 +1,21 @@
-const Categories = () => (
-  <section className="categories">
-    <p className="progress">Progress will be here</p>
-    <button type="button">Check Status</button>
-  </section>
-);
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+  return (
+    <div className="categories">
+      <h1>Categories list</h1>
+      <button
+        type="button"
+        onClick={() => {
+          dispatch(checkStatus());
+        }}
+      >
+        Check Status
+      </button>
+    </div>
+  );
+};
```

Comment:
- [ ] It would be nice to consider accessing the `statusReducer` which is for categories so that you can display the **_under construction_** word when the `check status` button is clicked. 👍